docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Contributor: Sujith R Pillai <sujithrpillai@gmail.com>
+
+services:
+  toolbox:
+    image:  us-central1-docker.pkg.dev/database-toolbox/toolbox/toolbox:latest
```

Comment:
nit: Maybe add a TODO above this line recommending that the user pin to a version instead of using latest?